modules/snooker/src/test/ts/browser/resize/ColumnSizesTest.ts

```diff
@@ -17,6 +18,8 @@ const pixelTableMissingWidthsHtml = `<table style="width: 400px; border-collapse
 </table>`;
 const tableWithSpansHtml = '<table style="width: 400px; border-collapse: collapse"><tbody><tr><td style="width: 400px;" colspan="2">A</td></tr></tbody></table>';
 const noneTableWithColsHtml = '<table><colgroup><col><col></colgroup><tbody><tr><td>A</td><td>A</td></tr></tbody></table>';
+const browser = PlatformDetection.detect().browser;
+const isChrome92 = browser.isChrome() && browser.version.major === 92;
```

Comment:
This needs to handle newer versions as well, otherwise we'll just run into this in a month again when Chrome 93 comes out.
```suggestion
const isChrome92 = browser.isChrome() && browser.version.major >= 92;
```

Reply:
Worth noting if we do this, it'll also let us know when Chrome fix the bug since the test will start failing again.